config/default.yml

```diff
@@ -2,9 +2,8 @@
 FactoryBot:
   Enabled: true
   Include:
-    - "**/spec/factories.rb"
+    - "**/factories.rb"
```

Comment:
I’m uncertain if it’s deliberate or not, but they would also load from factories/**/*.rb

Yet another doc https://github.com/thoughtbot/factory_bot/blob/cf3f21fcbbccbc40849c74a84eeb0bb0bd6c7606/docs/src/defining/file-paths.md?plain=1#L7 yhat doesn’t mention this consistently.